assets/src/edit-story/components/keyboard/index.js

```diff
@@ -39,6 +39,15 @@ const NON_EDITABLE_INPUT_TYPES = [
   'reset',
   'hidden',
 ];
+const CLICKABLE_INPUT_TYPES = [
```

Comment:
Should the `NON_EDITABLE_INPUT_TYPES` perhaps just be `[ ...CLICKABLE_INPUT_TYPES, 'hidden']`?

Reply:
Hm, not sure. Is it an either/or - either an input field is clickable or it's editable (or none of them, `hidden`)? Can that be considered a _universal truth_ about inputs?

Reply:
Looking further into this, a counter-example would be `<input type="date" />` - it seems to me to be both _clickable_ and _editable_.

So I'll leave this as-is for now.